### PR TITLE
[DOC] Remove `using namespace seqan3` from tutorial

### LINKED ISSUE #1364: Remove "using namespace seqan3;" from DOC - Tutorial
Chapters:
- [x] First steps
- [x] Parsing command line arguments
- [x] C++ Concepts
- [x] Alphabets
- [x] Ranges
- [x] Sequence file I/O
- [x] Computing pairwise alignments
- [x] Index and Search
- [x] Alignment file I/O
- [x] Implement your own read mapper

### PR DESCRIPTION
**Pull request by Irallia**
Resolves #1364

Additionally, the includes got sorted and std::endl got removed.

 The file doc/tutorial/alignment_file/alignment_file_solution2.cpp will be done by #1412.

### CODE REVIEW

**Comment by smehringer** [comment]
Please assign me and Rene (and marcel regarding the alignment module) as second reviewer and someone else as first reviewer :)

File: doc/tutorial/alignment_file/alignment_file_read_cigar.cpp [hunk 1]
```diff
@@ -30,15 +30,13 @@ write_file_dummy_struct go{};
...
-using namespace seqan3;
-
 int main()
 {
     std::filesystem::path tmp_dir = std::filesystem::temp_directory_path(); // get the temp directory
 
-    alignment_file_input fin{tmp_dir/"my.sam", fields<field::CIGAR>{}};
+    seqan3::alignment_file_input fin{tmp_dir/"my.sam", seqan3::fields<seqan3::field::CIGAR>{}};
 
     for (auto & [cigar] : fin)
```
Comment (smehringer) [inline doc/tutorial/alignment_file/alignment_file_read_cigar.cpp]: Can you rename the variable `cigar` to `cigar_vector? Do this in a separate commit. It will fix #1344 :) You can put `fixes  #1344` int he commit message, then it is linked.

File: doc/tutorial/pairwise_alignment/pa_assignment_3_solution.cpp [hunk 2]
```diff
@@ -27,10 +27,12 @@ int main()
     }
 
     // Configure the alignment kernel.
-    auto config = align_cfg::mode{global_alignment} |
-                  align_cfg::scoring{aminoacid_scoring_scheme{aminoacid_similarity_matrix::BLOSUM62}} |
-                  align_cfg::aligned_ends{free_ends_second};
+    auto config = seqan3::align_cfg::mode{seqan3::global_alignment} |
+                  seqan3::align_cfg::scoring{seqan3::aminoacid_scoring_scheme{
+                      seqan3::aminoacid_similarity_matrix::BLOSUM62
+                  }} |
```
Comment (joergi-w) [inline doc/tutorial/pairwise_alignment/pa_assignment_3_solution.cpp]: Can you move these to the end of the previous line because the matching `{}` are anyway not in the same column?

File: doc/tutorial/alignment_file/alignment_file_solution1.cpp [hunk 3]
```diff
@@ -29,22 +29,23 @@ write_file_dummy_struct go{};
 #include <seqan3/core/debug_stream.hpp>
 #include <seqan3/io/alignment_file/all.hpp>
 #include <seqan3/std/filesystem>
-#include <seqan3/range/views/get.hpp>
 #include <seqan3/std/ranges>
-
-using namespace seqan3;
+#include <seqan3/range/views/get.hpp>
```
Comment (joergi-w) [inline doc/tutorial/alignment_file/alignment_file_solution1.cpp]: please sort

File: doc/tutorial/pairwise_alignment/pairwise_alignment_solution_3.cpp [hunk 4]
```diff
@@ -6,18 +6,20 @@
...
     auto seq2 = "AFLPGWQEENKLSKIWMKDCGCLW"_aa27;
 
     // Configure the alignment kernel.
-    auto config = align_cfg::mode{global_alignment} |
-                  align_cfg::scoring{aminoacid_scoring_scheme{aminoacid_similarity_matrix::BLOSUM62}} |
-                  align_cfg::gap{gap_scheme{gap_score{-2}, gap_open_score{-9}}};
+    auto config = seqan3::align_cfg::mode{seqan3::global_alignment} |
+                  seqan3::align_cfg::scoring{seqan3::aminoacid_scoring_scheme{
+                      seqan3::aminoacid_similarity_matrix::BLOSUM62
+                  }} |
```
Comment (joergi-w) [inline doc/tutorial/pairwise_alignment/pairwise_alignment_solution_3.cpp]: also here... (and the other solution files)

File: doc/tutorial/sequence_file/sequence_file_solution3.cpp [hunk 5]
```diff
@@ -57,36 +57,34 @@ write_file_dummy_struct go{};
...
     // But you can also do this to retrieve all IDs into a vector:
     std::vector<std::string> ids = fin
-                                 | length_filter                                // apply length filter
-                                 | std::views::take(2)                          // take first two records
-                                 | views::get<field::ID>                        // select only ID from record
-                                 | views::convert<std::string &&>               // mark ID to be moved out of record
-                                 | views::to<std::vector<std::string>>;         // convert to container
+                                 | length_filter                                    // apply length filter
+                                 | std::views::take(2)                              // take first two records
+                                 | seqan3::views::get<seqan3::field::ID>    // select only ID from record
```
Comment (joergi-w) [inline doc/tutorial/sequence_file/sequence_file_solution3.cpp]: align this comment as well?

File: doc/tutorial/alignment_file/alignment_file_snippets.cpp [hunk 6]
```diff
@@ -89,13 +87,15 @@ int main()
 //![read_custom_fields]
     auto filename = std::filesystem::temp_directory_path()/"example.sam";
 
-    alignment_file_input fin{filename, fields<field::ID, field::SEQ, field::FLAG>{}};
+    seqan3::alignment_file_input fin{filename, seqan3::fields<seqan3::field::ID,
+                                               seqan3::field::SEQ,
```
Comment (smehringer) [inline doc/tutorial/alignment_file/alignment_file_snippets.cpp]: fix the indentation such that all `field::`'s are above each other.

File: doc/tutorial/alphabet/alphabet_gc_content.cpp [hunk 7]
```diff
@@ -9,7 +9,7 @@
 #include <seqan3/range/views/char_to.hpp>
 #include <seqan3/range/views/to.hpp>
 
-using namespace seqan3;
+using seqan3::operator""_dna5;
```
[the remark this answers is not in the document]
Comment (smehringer) [reply doc/tutorial/alphabet/alphabet_gc_content.cpp]: We should introduce a literals namespace that can be incuded. So fot the PR you can leave the operators whereever they are :)

File: doc/tutorial/argument_parser/disable_version_check.cpp [hunk 8]
```diff
@@ -1,9 +1,7 @@
 #include <seqan3/argument_parser/all.hpp>
 
-using namespace seqan3;
-
 int main(int argc, char ** argv)
 {
-    argument_parser myparser{"Game-of-Parsing", argc, argv, false};
+    seqan3::argument_parser myparser{"Game-of-Parsing", argc, argv, false};
     // disable version checks permanently --------------------^
```
Comment (smehringer) [inline doc/tutorial/argument_parser/disable_version_check.cpp]: add some more `----` such that the small "arrow" points to the `false` parameter :nail_care:
